src/lang/english.txt

```diff
@@ -265,7 +265,7 @@ STR_SHOW_HIDDEN_ENGINES_VEHICLE_AIRCRAFT_TOOLTIP                :{BLACK}By enabl
 STR_BUTTON_DEFAULT                                              :{BLACK}Default
 STR_BUTTON_CANCEL                                               :{BLACK}Cancel
 STR_BUTTON_OK                                                   :{BLACK}OK
-STR_WARNING_PASSWORD_SECURITY                                   :{YELLOW}Warning: Server administrators may be able to read any text entered here.
+STR_WARNING_PASSWORD_SECURITY                                   :{YELLOW}Warning: server administrators may be able to read any text entered here.
 
```

Comment:
This is wrong; after the colon is a complete sentence so should be capitalized.

Reply:
A colon isn't an end-of-sentence mark, just like a semi-colon. Usually capital letters are only used after colon if there's a paragraph break, although this is open to discussion.